app/scripts/contentscript.js

```diff
@@ -1,29 +1,19 @@
 const issueTitles = document.getElementsByClassName('js-issue-title') || [];
-const issuePattern = /([a-z]{2,}-\d+)(.*)/i;
+const issuePattern = /([a-z]{2,}-\d+)/gi;
 
 const updateNodes = () => {
-  for(let titleNode of issueTitles) {
-    const textContent = titleNode.textContent;
-
-    const matches = textContent.match(issuePattern);
-
-    if(matches) {
-      titleNode.textContent = '';
-
-      const ticketNumber = matches[1];
-      const restOfText = matches[2];
-
-      const restOfTextNode = document.createTextNode(restOfText);
-
-      let jiraLinkNode = document.createElement('a');
-      jiraLinkNode.href = `https://virtru.atlassian.net/browse/${ticketNumber}`;
-      const ticketNumberNode = document.createTextNode(ticketNumber);
-      jiraLinkNode.appendChild(ticketNumberNode);
-
-      titleNode.appendChild(jiraLinkNode);
-      titleNode.appendChild(restOfTextNode);
-    }
+  for (let titleNode of issueTitles) {
+    titleNode.innerHTML = titleNode.textContent.replace(issuePattern, (ticketNumber) => (
+      `<a
+         target="_blank"
```

Comment:
I don't want to set the `target` -- I would rather be less prescriptive about how the link opens, if you want to get a new window or a new tab, hold Cmd or Shift.